src/state/action-creators.js

```diff
@@ -158,28 +190,44 @@ export const moveForward = (id: DraggableId): MoveForwardAction => ({
   payload: id,
 });
 
-export type CancelAction = {
-  type: 'CANCEL',
-  payload: DraggableId
+type CleanAction = {
```

Comment:
Separating out 'clean' from 'cancel'. Clean is driven by errors, cancel is driven by the user